airflow/www/views.py

```diff
@@ -2249,7 +2256,14 @@ def get_int_arg(value, default=0):
             (no_obj_dags, obj_dags)[bool(dagbag.get_dag(dag.dag_id))].append(dag)
 
         max_dt = pendulum.timezone('UTC').convert(dt.datetime(9999, 12, 31))
-        dags = list(sorted(obj_dags, key=lambda dag: dagbag.get_dag(dag.dag_id).get_next_run_date(pendulum.now('UTC')) or max_dt))
+
+        def ordering(dags):
+            last_dr = dags.get_last_dagrun_by_run_date(include_externally_triggered=True, include_incomplete=True)
+            next_dr = dags.get_next_run_date(pendulum.now('UTC')) or max_dt
+            is_running = last_dr.state != State.RUNNING if last_dr and last_dr.execution_date else 1
```

Comment:
Hmm, actually this should be called `is_not_running`?

```suggestion
            is_not_running = (not last_dr) or (last_dr.execution_date and last_dr.state != State.RUNNING) 
```